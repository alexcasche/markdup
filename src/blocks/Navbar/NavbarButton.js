import { Button } from 'elements'

const NavbarButton = Button.extend`
  display: flex;
  flex-direction: column;
  justify-content: center;
  width: 20px;
  height: 40px;
  padding: 0;
  background-color: transparent;
`

export default NavbarButton
