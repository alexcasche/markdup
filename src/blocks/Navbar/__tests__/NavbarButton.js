import NavbarButton from '../NavbarButton'

describe('NavbarButton Element', () => {
  it('renders correctly', () => {
    const tree = render(<NavbarButton>Submit</NavbarButton>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
