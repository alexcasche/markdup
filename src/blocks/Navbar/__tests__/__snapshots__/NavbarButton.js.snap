// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarButton Element renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border: none;
  outline: none;
  font-family: "IBM Plex Sans Condensed",Helvetica,Arial,sans-serif;
  font-weight: 500;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  text-transform: uppercase;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 0.15s ease-in-out;
  transition: all 0.15s ease-in-out;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 20px;
  height: 40px;
  padding: 0;
  background-color: transparent;
}

.c0:hover {
  cursor: pointer;
}

<button
  className="c0"
>
  Submit
</button>
`;
